data/clients.json

```diff
@@ -904,6 +904,20 @@
 			"url": "https://smallstep.com/cli/",
 			"acme_v2": "true",
 			"category": "Go"
+		},
+		{
+			"name": "CycloneACME",
+			"url": "https://oryx-embedded.com/products/CycloneACME",
+			"category": "C",
+			"acme_v2": "true",
+			"library": "CycloneACME",
```

Comment:
The build is failing because the `library` field is a cross-reference to the `libraries` list on line 27, and its value has to match one of the items there.
```suggestion
```